src/lib/components/ProjectsPage/ProjectsTable.tsx

```diff
@@ -56,9 +58,22 @@ export const ProjectsTable = ({ projectType }: IProjectPageProps) => {
     setActiveFilters({ ...activeFilterFields });
   }, [watchedFields]);
 
+  const filterBySearch = (row: any) => {
```

Comment:
move this function outside the projectsTable to make it reusable for other table as well...

Reply:
it might look like this:
```ts
  const filterBySearch = <T extends {}, F extends keyof T>(
    row: T,
    searchBy: F[],
    searchValue: string,
  ) =>
    searchValue.trim().length > 0
      ? searchBy.some((searchField) =>
          String(row[searchField]).includes(searchValue),
        )
      : true;
```

Reply:
done!